skel/.vimrc

```diff
@@ -0,0 +1,18 @@
+set number
+set smartcase
+set autoindent
+set nowrap
+set incsearch
+set hlsearch
+set cursorline
+set whichwrap=b,s,<,>,[,]
+set backspace=2
+set ruler
+set laststatus=2
+set noshowcmd
+set ttyfast
+colo elflord
+syntax on
+hi CursorLine	cterm=NONE	ctermbg=236	ctermfg=NONE
```

Comment:
All hi lines will go bad optically depending on the colourscheme and terminal combination. This should be done only be the colourscheme, very much so in a global configuration.

Reply:
Sorry, I should have been clearer: I submitted this PR to address one specific problem:

With our current urxvt & terminator colourscheme vim becomes almost unreadable if syntax highlighting is applied.

The *elflord* colourscheme corrects this (IMO) and greatly improves the contrast and readability so that is the only modification that I would consider to be important.

I do appreciate though that neither urxvt or vim are included in BL and syntax highlighting remains off until enabled so perhaps this should be closed.

Reply:
I like elflord myself on systems I'm usually never on. Perhaps, as I suggested, be could ship a vimrc.local which enables some essential options already commented in the default vimrc plus elflord? For example, `filetype plugin indent on` as well as `syntax on` are things which vim on the desktop really should have by default.